src/main/java/reposense/util/FileUtil.java

```diff
@@ -95,8 +95,9 @@ public static void zipFoldersAndFiles(List<Path> pathsToZip,
 
```

Comment:
Can move `sourcePath` and `outputPath` variables for the `zipFoldersAndFiles` function to line 73, and keep only `fileTypes` in line 74.